src/style/properties.js

```diff
@@ -759,7 +760,7 @@ export class Properties<Props: Object> {
             this.defaultTransitioningPropertyValues[property] =
                 defaultTransitionablePropertyValue.untransitioned();
             this.defaultPossiblyEvaluatedValues[property] =
-                defaultPropertyValue.possiblyEvaluate(({}: any));
+                defaultPropertyValue.possiblyEvaluate(defaultParameters);
```

Comment:
Is this related to the atmosphere code?

Reply:
This was necessary to fix an issue that manifested itself now that we introduced default parameters in a root style specification entry. Having a dummy parameter at creation allows further dynamic evaluation of the property values.

Reply:
okay thanks for the clarification! :+1: